opm/input/eclipse/Schedule/RSTConfig.cpp

```diff
@@ -194,6 +194,50 @@ bool is_RPTRST_mnemonic(const std::string& kw)
     return std::binary_search(std::begin(valid), std::end(valid), kw);
 }
 
+bool is_RPTRST_mnemonic_compositional(const std::string& kw)
+{
+    // Every compositional keyword we want to not simply ignore when handling
+    // RPTRST.  The list is sorted, so we can use binary_search for log(n)
+    // lookup.  It is important that the list is sorted, but these are all
+    // the keywords listed in the manual and unlikely to change at all
+    static constexpr const char* valid[] = {
+        "AIM",      "ALSURF",   "ALSTML",   "AMF",      "AQSP",    "AQPH",
```

Comment:
Not sure how we should handle `FLOCn` and `LGLCn`, where `n`  is the component number and an integer, 
and also the  `TRMFxxxx` and `xxxx` is the tracer name and string.

I copied FLORES- from the `is_RPTRST_mnemonic`  without knowing why.

Reply:
> I copied `FLORES-` from the `is_RPTRST_mnemonic` without knowing the rationale behind the `FLORES-`.

`FLORES-` is an OPM extension introduced in #3823.

Reply:
> Not sure how we should handle `FLOCn` and `LGLCn`, where `n`

We may need to pull the arrays whose names include a component number into a separate list.  I haven't given much thought to this problem yet.

Reply:
Then I think I should NOT have FLOC- , LGLC- and TRMF- in the list. And very likely `FLOCn`, `LGLCn` and `TRMFXXXX` needs to be done differently, and they can be out of this list.  Anyway, I will wait for your suggestion and just put a note here for later record.

Reply:
>>> I copied `FLORES-` from the `is_RPTRST_mnemonic` without knowing the rationale behind the `FLORES-`.
>> 
>> `FLORES-` is an OPM extension introduced in #3823.
>
> Then I think I should NOT have `FLOC-`, `LGLC-`, or `TRMF-` in the list.

I think I agree.